R/cloglognormal.R

```diff
@@ -94,16 +94,13 @@ posterior_epred_cloglognormal <- function(prep) {
 #'
 #' @examples # Running the example might take a while and may make RStudio unresponsive.
```

Comment:
Please remove these kinds of off-hand comments throughout the project. They are fun but add clutter to the documentation.

Reply:
I will shorten them. Two lines seem excessive. But beeing informed about this is important, so I would suggest, not to remove them completely.

Reply:
I don't think they are necessary. They obviously fit a model which takes some time.

Reply:
That is correct. I thought to inform, because those examples will also make the RStudio console unresponsive. This behavior did seem odd to me, given I was unaware about the relation.
But I will remove them, given that behavior is not a big deal.